From 2f6ea1b82053fabf20b89d9b44288ad3bf84b35e Mon Sep 17 00:00:00 2001
From: Jingoo Han <jg1.han@samsung.com>
Date: Tue, 30 Jul 2013 17:08:05 +0900
Subject: gpio: use dev_get_platdata()

Use the wrapper function for retrieving the platform data instead of
accessing dev->platform_data directly.

Signed-off-by: Jingoo Han <jg1.han@samsung.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
(cherry picked from commit e56aee1897fd27631c1cb28e12b0fb8f8f9736f7)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>

Conflicts:
	drivers/gpio/gpio-74x164.c
	drivers/gpio/gpio-adp5520.c
	drivers/gpio/gpio-adp5588.c
	drivers/gpio/gpio-arizona.c
	drivers/gpio/gpio-da9052.c
	drivers/gpio/gpio-da9055.c
	drivers/gpio/gpio-ich.c
	drivers/gpio/gpio-janz-ttl.c
	drivers/gpio/gpio-kempld.c
	drivers/gpio/gpio-max730x.c
	drivers/gpio/gpio-max732x.c
	drivers/gpio/gpio-mc33880.c
	drivers/gpio/gpio-mcp23s08.c
	drivers/gpio/gpio-msic.c
	drivers/gpio/gpio-omap.c
	drivers/gpio/gpio-pca953x.c
	drivers/gpio/gpio-pcf857x.c
	drivers/gpio/gpio-pl061.c
	drivers/gpio/gpio-rcar.c
	drivers/gpio/gpio-rdc321x.c
	drivers/gpio/gpio-sta2x11.c
	drivers/gpio/gpio-sx150x.c
	drivers/gpio/gpio-timberdale.c
	drivers/gpio/gpio-tps65912.c
	drivers/gpio/gpio-ts5500.c
	drivers/gpio/gpio-twl4030.c
	drivers/gpio/gpio-ucb1400.c
	drivers/gpio/gpio-wm831x.c
	drivers/gpio/gpio-wm8350.c
	drivers/gpio/gpio-wm8994.c
---
 drivers/gpio/gpio-em.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/gpio-em.c b/drivers/gpio/gpio-em.c
index 5cba855638bf..847987c4f04a 100644
--- a/drivers/gpio/gpio-em.c
+++ b/drivers/gpio/gpio-em.c
@@ -237,7 +237,7 @@ static struct irq_domain_ops em_gio_irq_domain_ops = {
 static int em_gio_probe(struct platform_device *pdev)
 {
 	struct gpio_em_config pdata_dt;
-	struct gpio_em_config *pdata = pdev->dev.platform_data;
+	struct gpio_em_config *pdata = dev_get_platdata(&pdev->dev);
 	struct em_gio_priv *p;
 	struct resource *io[2], *irq[2];
 	struct gpio_chip *gpio_chip;
-- 
1.8.5.rc3

